package/MDAnalysis/lib/trans.pyx

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+cimport cython
+cimport numpy as np
+
+
+@cython.boundscheck(False)
+@cython.wraparound(False)
+def residue_positions(
+        float[:, :] coords not None,
```

Comment:
I would keep your current version as this is less pain for someone who uses the function.